base/abstractarray.jl

```diff
@@ -1255,7 +1255,7 @@ end
     getindex(A, inds...)
 
 Return a subset of array `A` as specified by `inds`, where each `ind` may be,
```

Comment:
```suggestion
Return a subset of array `A` as selected by the indices `inds`

Each index may be any [supported index type](@ref man-supported-index-types), such as
an `Int`, an [`AbstractRange`](@ref), a [`Vector`](@ref), or a [`CartesianIndex`](@ref).

For more details, see the entire manual section on [array indexing](@ref man-array-indexing).
```

Reply:
This is a great addition — I want to go one step farther! :) What do you think of something like the above?

Reply:
The two "or"s read weird to me. (Is there a way to make a new suggested based on your suggestion?)

Probably should also add `BitArray` with an example like `A[A.>2]`. 

Also, it looks like again I didn't grab the latest `master` version somehow, because there is a conflict below with information about `view`s.

Reply:
The whole list of supported indices is here: https://docs.julialang.org/en/v1/manual/arrays/#man-supported-index-types

That's really what I want to point first and foremost.  Everything else is just the most-commonly-used examples.  I'd probably rather include information about `:` before bitarrays, I think.

Reply:
This commit messed up somehow.

Reply:
Yeah, online suggestions can only do so much... I knew it would need some fixups afterwards. Thanks for doing it! :)